controllers/topics/index.js

```diff
@@ -103,10 +103,15 @@ const putFollowTopic = async (req, res) => {
 };
 
 const getTopics = async (req, res) => {
-  const {name} = req.query;
+  const {name, withMinimumFollower = true} = req.query;
   const signUserId = req.userId;
   const anonymousId = await getAnonymUser(req.userId);
 
+  let withMinimumFollowerQuery = '';
+  if (withMinimumFollower) {
+    withMinimumFollowerQuery = `HAVING count("topicFollower"."user_id") >= 5`;
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `withMinimumFollower` query parameter is expected to be a boolean based on the PR summary, but it's not being treated as such in the code. It's only checking if `withMinimumFollower` exists and not whether it's true or false. This could lead to unexpected behavior if a user sets `withMinimumFollower` to false but still gets results filtered by minimum followers. You should explicitly check if `withMinimumFollower` is true.
```diff
-  if (withMinimumFollower) {
+  if (withMinimumFollower === 'true') {
     withMinimumFollowerQuery = `HAVING count("topicFollower"."user_id") >= 5`;
   }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->